### DISCUSSION TITLE
Add ffprobe for additional media metadata

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3064] [Feature Requests]
https://www.npmjs.com/package/ffprobe

**Comment by rijkvanzanten** [comment]
Lets rely on https://www.npmjs.com/package/fluent-ffmpeg instead

Also needs to handle for the case where ffprobe/ffmpeg doesn't exist on the machine

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
